packages/payload/src/queues/utilities/updateJob.ts

```diff
@@ -0,0 +1,156 @@
+import type { PayloadRequest, Where } from '../../types/index.js'
+import type { BaseJob } from '../config/types/workflowTypes.js'
+
+import { jobAfterRead, jobsCollectionSlug } from '../config/index.js'
+import { sanitizeUpdateData } from './sanitizeUpdateData.js'
+
+type RunJobArgs = {
+  data: Partial<BaseJob>
+  depth?: number
+  disableTransaction?: boolean
+  id: number | string
+  req: PayloadRequest
+  returning?: boolean
+}
+export async function updateJob({
```

Comment:
This appears to be completely redundant. I would only have updateJobs unless I'm missing something.

Reply:
OR make updateJob use updateJobs if your really want to. I hate having duplicate code to maintain.

Reply:
See https://github.com/payloadcms/payload/pull/11489#discussion_r2003864257